src/app/static/src/app/components/outputTable/TableData.vue

```diff
@@ -8,10 +11,19 @@ import { RootState } from '../../root';
 import { useStore } from 'vuex';
 import { Filter } from '../../generated';
 import TableReshapeData from './TableReshapeData.vue';
+import DownloadButton from '../downloadIndicator/DownloadButton.vue';
+import { exportService } from '../../dataExportService';
+import { appendCurrentDateTime } from '../../utils';
+
+// defines the order of headers on the excel download
+const header = [ "area_id", "area_name", "area_level",
```

Comment:
Again, it feels like this could maybe be metadata'ed a bit, but I think that's something we could add later. 
They want 'parent area id' - just the next level up, not the whole hierarchy?

Reply:
Maybe we need some layer where "business" logic lives in hint? I don't think we need to necessarily get this back from hintr if it isn't something science want to change. But putting it outside of the display component is usually good practice right? I have similar thoughts on the missing data PR where I'm putting some business-like logic into the Plotly component

Reply:
hmm not sure we want this in the metadata, i guess we could put it there but its just about the ordering of the excel sheet, i think maybe we should leave it in some random file because as rob said this isnt something that will change so can just be hardcoded in. Perhaps something to think about for the plot cleanup?

The parent_area_id was a think rachel asked, yh she didnt want the whole hierarchy, not sure why honestly haha

Reply:
> Maybe we need some layer where "business" logic lives in hint? I don't think we need to necessarily get this back from hintr if it isn't something science want to change. But putting it outside of the display component is usually good practice right? I have similar thoughts on the missing data PR where I'm putting some business-like logic into the Plotly component

I guess that's the idea of the 'service' layer, though not sure we always think of them 'services' exactly.